### ISSUE TITLE
not pulling in node_helper

### ISSUE THREAD

**Issue by JPMinAR** [bastilimbach/docker-MagicMirror#29]
So took me a little bit to determine what was happening but since I have a 'modules' directory mapped to my docker instance it does not include the /modules/node_modules/node_helper/index.js dependency that multiple applications assume to be present.

Can the fact that this needs to be included in the mapped modules instance or could this be conditionally copied over as part of the 'else' condition of the 'docker-entrypoint.sh' so as to eliminate confusion from user that are not aware of this required dependency.

**Comment by sdetweil** [comment]
that node module is included in the default git clone install as part of the base. 

I have submitted an update to the master MagicMirror repo to load this node_helper.js class module differently, so it will be present as part of npm install
I hope that it will be accepted in version 2.10

**Comment by bastilimbach** [comment]
@sdetweil Is this solved?

**Comment by sdetweil** [comment]
Yes. It is located in the js folder now, with an alias in package.json.

So u need both changes
